library/x509_crt.c

```diff
@@ -820,7 +820,17 @@ static int x509_get_crt_ext( unsigned char **p,
             break;
 
         default:
-            return( MBEDTLS_ERR_X509_FEATURE_UNAVAILABLE );
+            /*
+             * In case this is a non critical extension, which the oid layer
+             * support, but there isn't an x509 parser for it,
```

Comment:
English:
```diff
-             * In case this is a non critical extension, which the oid layer
-             * support, but there isn't an x509 parser for it,
+             * If this is a non-critical extension, which the oid layer
+             * supports, but there isn't an x509 parser for it,
```